### PR TITLE
Added: Faker.Name.Es and Faker.Name.PtBr

### PR DESCRIPTION
**Pull request by vbrazo**
I've added:

- [x] USAGE.md docs
- [x] CHANGELOG.md
- [x] `Faker.Name.En` doctests
- [x] Spanish translations for Faker.Name API - `Faker.Name.Es`
- [x] Brazilian Portuguese translations for Faker.Name API - `Faker.Name.PtBr`
- [x] `Faker.Name` doctests

### CODE REVIEW

File: test/faker/name/pt_br_test.exs [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  end
+
+  test "prefix/0" do
+    assert is_binary(Faker.Name.prefix)
+  end
+
+  test "suffix/0" do
+    assert is_binary(Faker.Name.suffix)
+  end
+end
```
Comment (tobyhinloopen) [inline test/faker/name/pt_br_test.exs]: Note: Your editor seems to not append New-Line to EOF automagically :) For some reason, you should always finish a file with a New-Line.

**Comment by tobyhinloopen** [comment]
LGTM :) 

Related to newlines at end of file:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by vbrazo** [comment]
@tobyhinloopen I had to reconfigure my editor. Sorry about that. 

I just pushed the changes that you requested. Thanks.

File: lib/faker/name/es.ex [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
..."Alejandra", "Alicia", "Amalia", "Ana", "Ana Luisa", "Ana María", "Andrea", "Anita", "Ángela", "Antonia", "Ariadna", "Barbara", "Beatriz", "Berta", "Blanca", "Caridad", Carla, Carlota, Carmen, Carolina, Catalina, Cecilia, Clara, Claudia, "Concepción", "Conchita", "Cristina", "Daniela", "Débora", "Diana", "Dolores", "Lola", "Dorotea", "Elena", "Elisa", "Eloisa", "Elsa", "Elvira", "Emilia", "Esperanza", "Estela", "Ester", "Eva", "Florencia", "Francisca", "Gabriela", "Gloria", "Graciela", "Guadalupe", "Guillermina", "Inés", "Irene", "Isabel", "Isabela", "Josefina", "Juana", "Julia", "Laura", "Leonor", "Leticia", "Lilia", "Lorena", "Lourdes", "Lucia", "Luisa", "Luz", "Magdalena", "Manuela", "Marcela", "Margarita", "María", "María del Carmen", "María Cristina", "María Elena", "María Eugenia", "María José", "María Luisa", "María Soledad", "María Teresa", "Mariana", "Maricarmen", "Marilu", "Marisol", "Marta", "Mayte", "Mercedes", "Micaela", "Mónica", "Natalia", "Norma", "Olivia", "Patricia", "Pilar", "Ramona", "Raquel", "Rebeca", "Reina", "Rocio", "Rosa", "Rosalia", "Rosario", "Sara", "Silvia", "Sofia", "Soledad", "Sonia", "Susana", "Teresa", "Verónica", "Victoria", "Virginia", "Yolanda"]
```
Comment (igas) [inline lib/faker/name/es.ex]: please use elixir formatter. I'm slowly changing codebase to it. (don't want to have a commit just for this against whole project)

File: test/faker/name/es_test.exs [hunk 3]
```diff
@@ -0,0 +1,23 @@
+defmodule Faker.Name.EsTest do
+  use ExUnit.Case, async: true
+  
+  test "first_name/0" do
```
Comment (igas) [inline test/faker/name/es_test.exs]: Please use doctests like in other PR.

**Comment by vbrazo** [comment]
@igas I've just added the doctests for the new locales 👍

File: test/faker/name_test.exs [hunk 4]
```diff
@@ -1,27 +1,31 @@
...
+defmodule Faker.NameTest do
   use ExUnit.Case, async: true
 
+  doctest Faker.Name.En
+  doctest Faker.Name.Es
+  doctest Faker.Name.PtBr
+
   test "first_name/0" do
-    assert is_binary(Faker.Name.first_name)
+    assert is_binary(Faker.Name.first_name())
```
Comment (vbrazo) [inline test/faker/name_test.exs]: I forgot to convert these binary tests to doctests and just declare `doctest Faker.Name`. I'll do that tomorrow 👍

**Comment by vbrazo** [comment]
@igas I'm back, and I just finished the changes that you requested in the PRs a few days ago. Thanks for reviewing :100:

**Pull request merged**